modules/javafx.controls/src/main/java/javafx/scene/control/skin/TableHeaderRow.java

```diff
@@ -475,7 +464,26 @@ protected NestedTableColumnHeader createRootHeader() {
         return new NestedTableColumnHeader(null);
     }
 
-
+    /**
+     * Shows a menu containing all leaf columns as item.
+     * An item can be selected/deselected to make the corresponding column visible/invisible.
+     *
+     * @implNote This method can be overridden to create and show a custom menu.
+     * @param mouseEvent the {@link MouseEvent} which was generated when the table menu button was pressed.
```

Comment:
Minor: by convention, the text following `@param` and `@return` tags should not end with a period.

Reply:
Changed. And thanks for the hint, I actually didn't know that this is a convention. 
Personally, I also use the variant without a period, so it's good to know that this is indeed the convention. 😄